class.go

```diff
@@ -41,21 +44,29 @@ func (q HtbClassAttrs) String() string {
 // Htb class
 type HtbClass struct {
 	ClassAttrs
-	Rate    uint64
-	Ceil    uint64
+	Rate    nl.TcRateSpec
+	Ceil    nl.TcRateSpec
 	Buffer  uint32
 	Cbuffer uint32
 	Quantum uint32
 	Level   uint32
 	Prio    uint32
+	Rtab    [256]uint32
+	Ctab    [256]uint32
 }
 
-func NewHtbClass(attrs ClassAttrs, cattrs HtbClassAttrs) *HtbClass {
+func NewHtbClass(attrs ClassAttrs, cattrs HtbClassAttrs) (*HtbClass, error) {
```

Comment:
I think it would be best to change the `HtbClassAttrs` struct to make Rate and Ceil both `nl.TcRateSpec` that way the signature of this function can remain unchanged.